pages/quickstart.livemd

```diff
@@ -257,7 +253,7 @@ end
 #Ecto.Changeset<
   action: nil,
   changes: %{question: "What is the meaning of life?", answer: "Sex, drugs, and rock'n roll"},
-  errors: [answer: {"is invalid, Do not say anything objectionable", []}],
+  errors: [answer: {"is invalid, Contains objectionable content", []}],
```

Comment:
Some changes to this file look unrelated to the rest of the PR. Here the error message is based on line 230 https://github.com/thmsmlr/instructor_ex/pull/46/files#diff-dbcbac73b1730f428122f43c1c48a1614650f5eb3ebd5319ffdbe293ea937a3cR230